app/helpers/groups_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module GroupsHelper
+  def types
+    Group.distinct(:type).uniq.reverse
+  end
+
+  def references(type)
+    return Authority.authorized if type == 'AuthorityGroup'
+    return County.authorized if type == 'CountyGroup'
+    Instance.all
+  end
+end
```

Comment:
```suggestion
  def references(group)
    return Authority.authorized if group.is_a? AuthorityGroup
    return County.authorized if group.is_a? CountyGroup
    Instance.all
  end
end
```
Dann muss der Aufruf aber auch angepasst werden

Reply:
es ist nicht immer ein Gruppen-Objekt vorhanden. Beim rendern des partials im DataDependencies controller habe ich keine Gruppe von der ich das abfragen kann.

Reply:
Dann bleibt es so